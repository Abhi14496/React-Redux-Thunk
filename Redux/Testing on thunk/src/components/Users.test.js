import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Hello from './Users';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
configure({adapter: new Adapter()});
const mockStore = configureStore();
const store = mockStore()

const props = {};

describe('My users Component', () => {
    it('should render correctly with default props and redux store', () => {
        
        const wrapper = shallow(
            <Provider store={store}>
             <Hello {...props} />
            </Provider>
        );
        expect(wrapper).toMatchSnapshot();
   
    });
});