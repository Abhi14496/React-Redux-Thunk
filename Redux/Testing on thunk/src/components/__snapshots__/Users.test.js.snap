// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My users Component should render correctly with default props and redux store 1`] = `ShallowWrapper {}`;
